advanced/index.php

```diff
@@ -46,7 +46,9 @@
 // Set response header
 function setHeader($type = "x") {
     header("X-Pi-hole: A black hole for Internet advertisements.");
-    if (isset($type) && $type === "js") header("Content-Type: application/javascript");
+    if ($type === "js") {
+        header("Content-Type: application/javascript");
```

Comment:
BTW we use `text/javascript` but that's unrelated to this PR, I just noticed it :)

https://github.com/pi-hole/pi-hole/blob/6b536b7428a1f57ff34ddc444ded6d3a62b00a38/advanced/lighttpd.conf.debian#L72